src/couch_views/test/couch_views_indexer_test.erl

```diff
@@ -388,6 +388,142 @@ multipe_identical_keys_from_same_doc(Db) ->
         ], Out).
 
 
+handle_size_key_limits(Db) ->
+    ok = meck:new(config, [passthrough]),
+    ok = meck:expect(config, get_integer, fun(Section, _, Default) ->
+        case Section of
+            "couch_views" -> 15;
+            _ -> Default
+        end
+    end),
+
+    DDoc = create_ddoc(multi_emit_key_limit),
+    Docs = [doc(1)] ++ [doc(2)],
+
+    {ok, _} = fabric2_db:update_docs(Db, [DDoc | Docs], []),
+
+    {ok, Out} = couch_views:query(
+        Db,
+        DDoc,
+        <<"map_fun1">>,
+        fun fold_fun/2,
+        [],
+        #mrargs{}
+    ),
+
+    ?assertEqual([
+        {row, [
```

Comment:
If you want, to avoid repeating 5 lines for each row, could create a `row(Id, K, V)` function so then it would look like:

```?assertEqual(row(<<"1">>, 1, 1), Out)```

Reply:
Looks good